### DISCUSSION TITLE
Firmware version

### DISCUSSION THREAD

**Discussion by supperka** [roleoroleo/sonoff-hack#172] [Q&A]
Hello, just got this camera with firmware GK-200MP2D. 
I don't know if I can install this hack. Do you know anyone if Is safe? 
Thank you 👍 
![Messenger_creation_68743fd3-1e0f-4bd0-b823-d293c086640d](https://github.com/roleoroleo/sonoff-hack/assets/3536196/713e7777-6b38-4161-86fc-62bad520abf6)

**Comment by roleoroleo** [comment]
Check the fw revision, please.

**Comment by supperka** [comment]
I found 50520230320

**Reply by roleoroleo** [reply]
Sorry but your version is not supported at the moment.
I haven't got this model and I'm not able to add the support.
You can try if the hack for GK-200MP2C works.
This try will not brick the cam.

**Comment by supperka** [comment]
Thank you for your help, I already did. Doesn't work camera won't boot.

**Comment by roleoroleo** [comment]
Where did you buy it?
Could you send me a link?

**Comment by supperka** [comment]
From our local seller
https://m.alza.cz/sonoff-gk-200mp2-bcloud-storage-adapter-d7598985.htm
